components/connectivity-adapter/cmd/main.go

```diff
@@ -7,14 +7,14 @@ import (
 
 	"github.com/gorilla/mux"
 	"github.com/kyma-incubator/compass/components/connectivity-adapter/internal/appregistry"
-	connector "github.com/kyma-incubator/compass/components/connectivity-adapter/internal/connector"
+	connector "github.com/kyma-incubator/compass/components/connectivity-adapter/internal/connectorservice"
 	"github.com/kyma-incubator/compass/components/connectivity-adapter/pkg/health"
 	"github.com/pkg/errors"
 	"github.com/vrischmann/envconfig"
 )
 
 type config struct {
-	Address string `envconfig:"default=127.0.0.1:8080"`
+	APIAddress string `envconfig:"default=127.0.0.1:8080"`
```

Comment:
Do we need to rename it? Then we would need to update docs as well and helm charts values

Reply:
Done